tests/unittests/data/components.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+study:
+  nodes:
+    - id: N
+      model: node
+
+  components:
+    - id: G
+      model: generator
+      parameters:
+        - name: cost
+          type: constant
```

Comment:
Here we can use a "timeseries" type to check that it will fetch data from the associated timeseries file.

(Or use another test file for this, but we shoudl test this behaviour)